__test__/getAllProducts.test.js

```diff
@@ -0,0 +1,41 @@
+const request = require('supertest');
+const app = require('../server/app');
+const connection = require('../server/database/connection');
+const buildDb = require('../server/database/data/build');
+const { getAllProducts } = require('../server/database/quieres');
+
+describe('Test Routes, Test Database getClients query', () => {
+  beforeEach(() => buildDb());
+  afterAll(() => connection.end());
+
+  test('getAllProducts >> This test will return clients data', () => getAllProducts()
```

Comment:
please modify describe and test texts, remove clients and getClients.

Reply:
done